src/app/vault/share.component.ts

```diff
@@ -15,10 +16,10 @@ import { ShareComponent as BaseShareComponent } from 'jslib/angular/components/s
 export class ShareComponent extends BaseShareComponent {
     selectedCollectionId: string = undefined;
 
-    constructor(cipherService: CipherService, i18nService: I18nService,
+    constructor(private localCipherService: CipherService, i18nService: I18nService,
```

Comment:
Curious about the reason to make it private

Reply:
The one from BaseShareComponent is private so I cannot access this.cipherService

So I had to declare a new `localCipherService` (it is not possible to use the same name) and inject it in the `super` constructor

In typescript, a constructor argument is scoped to constructor until you add a modifier (public, private or protected) then it becomes class scoped. There is no sens here to make it public, so I should use private or protected. I can make it protected, I have no preference here.